src/foam/nanos/approval/ApprovalRequest.js

```diff
@@ -64,19 +64,12 @@
 
   sections: [
     {
-      name: 'requestDetails',
-      title: 'Approval Request Information',
-      order: 0
-    },
-    {
-      name: 'complianceInformation',
```

Comment:
Need to update this section reenaming for properties on ComplianceApprovalRequest np side I believe btw

Reply:
I'm have a PR ready for that.